src/main/generic/Core.js

```diff
@@ -1,9 +1,9 @@
 class Core {
-    constructor() {
-        return this._init();
+    constructor(options) {
+        return this._init(options);
     }
 
-    async _init() {
+    async _init({ walletSeed }) {
```

Comment:
Thanks for the reference. It looks like this is not supported by all major browsers.
So, I suggest to make the unpacking more explicit.